.github/workflows/publish_package.yml

```diff
@@ -0,0 +1,32 @@
+name: "Publishing Package"
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - run: npm install --global yarn
```

Comment:
Does this persist across jobs? Should we specify a version here to make sure it matches our version in package.json